service/src/main/kotlin/io/provenance/explorer/grpc/flow/FlowApiGrpcClient.kt

```diff
@@ -32,8 +32,8 @@ class FlowApiGrpcClient(flowApiChannelUri: URI) {
                     }
                 }
                 .idleTimeout(60, TimeUnit.SECONDS)
-                .keepAliveTime(10, TimeUnit.SECONDS)
-                .keepAliveTimeout(10, TimeUnit.SECONDS)
+                .keepAliveTime(300, TimeUnit.SECONDS)
+                .keepAliveTimeout(300, TimeUnit.SECONDS)
```

Comment:
These are pretty long. It's been a while since I looked at these, but I think this means that a keep alive ping will wait 5 minutes before considering it a failure and establishing a fresh connection. I believe this means that in the worst possible case, explorer will fail all grpc calls to the server for 10 minutes (5 + 5).

Is it possible the server side is actually the one with a min keep alive ping setting that's too high?

Reply:
Yes, this is too high.  

I went and checked what the default that the  grpc lib is using which is 15s.  The original configuration of 10 seconds was a copy and past from the other explorer grpc clients that call the blockchain node.

I think changing these to 15s in the explorer-service is a better solution than changing the defaults on the flow-api?

Reply:
👍 anything under 20-30s sounds reasonable to me. I think it will still come down to what the server side (flow-api) has set for the min keepalive because it would have to be something less than whatever value is being set here.

Reply:
Yep, I figure the default set by the grpc library (15s) was chosen as a reasonable time.  If we need to adjust, we can maybe make it easily configurable on both ends with config files.

Reply:
For reference it seems like this is the go grpc server side defaults https://pkg.go.dev/google.golang.org/grpc/keepalive#EnforcementPolicy. Unfortunately I don't think the defaults are in the grpc spec so languages can default independently.